pkg/asset/manifests/openstack/cloudproviderconfig.go

```diff
@@ -17,6 +17,10 @@ secret-namespace = kube-system
 		res += "region = " + cloud.RegionName + "\n"
 	}
 
+	if cloud.CACertFile != "" {
+		res += "ca-file = ca-bundle.pem\n"
```

Comment:
This shoud be an absolute path.
How about `/etc/pki/cloud-provider/ca-cert.pem` ?

Reply:
I was kinda hoping that it would look in its current working directory first, but we can also use https://github.com/openshift/library-go/blob/e560b33797f5dcde5d0e7b567e371a741e368543/pkg/operator/configobserver/cloudprovider/observe_cloudprovider.go#L20 as the absolute path.